### DISCUSSION TITLE
How do I set token and config if I want to use both device priority mode and custom mode?

### DISCUSSION THREAD

**Discussion by Han-sangwook** [chakra-ui/panda#2817] [Q&A] [answered]
How do I set token and config if I want to use both device priority mode and custom mode?

ex ) dark / light / device

```
	primaryNormal: {
		value: {
			base: blue[400],
			_dark: blue[400],
             _osDark: blue[400], // ?????
		},
	},
```

**Comment by segunadebayo** [comment, marked as answer]
I would recommend you keep it to just base and _dark, then use a library like next-themes or remix-themes to resolve color scheme to either light/dark.
